src/main/java/org/testobject/espressorunner/Configuration.java

```diff
@@ -18,7 +18,7 @@
 	@Parameter(names = "--username", description = "Your TestObject username", required = true, order = 2)
 	private String username;
 
-	@Parameter(names = "--password", description = "Your TestObject password", required = true, order = 3)
```

Comment:
neither `apiKey` nor `password` has `required = true` but at least one is required. do we enforce that anywhere else?